ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -164,12 +164,12 @@ case class CommandSubmitter(
 
     }
 
-    val generator = new CommandGenerator(
+    val generator = CommandGenerator(
       randomnessProvider = RandomnessProvider.Default,
       signatory = signatory,
       config = config,
       observers = observers,
-    )
+    ).asInstanceOf[FooCommandGenerator]
```

Comment:
```suggestion
    )
```

Reply:
Yes, I've fixed this on the other branch, will do same here.